wled00/FX.h

```diff
@@ -962,6 +972,13 @@ class WS2812FX {  // 96 bytes
       setUpSegmentFromQueuedChanges(void);
 };
 
+inline uint32_t hashInt(uint32_t s) {
```

Comment:
Please move this function into `util.cpp` as it may be used in the same manner as `crc16()`